src/bit-systems/media-loading.ts

```diff
@@ -216,21 +217,20 @@ function* loadAndAnimateMedia(world: HubsWorld, eid: EntityID, clearRollbacks: C
   if (MediaLoader.flags[eid] & MEDIA_LOADER_FLAGS.ANIMATE_LOAD) {
     yield* animateScale(world, media);
   }
-  removeComponent(world, MediaLoader, eid);
 
-  if (media) {
-    if (hasComponent(world, MediaLoaded, media)) {
-      const box = getBox(world, eid, media);
-      addComponent(world, MediaContentBounds, eid);
-      box.getSize(tmpVector);
-      MediaContentBounds.bounds[eid].set(tmpVector.toArray());
-    }
-    // TODO update scale?
-    inflatePhysicsShape(world, eid, {
-      type: hasComponent(world, GLTFModel, media) ? Shape.HULL : Shape.BOX,
-      minHalfExtent: 0.04
-    });
-  }
+  const box = getBox(world, eid, media);
+  addComponent(world, MediaContentBounds, eid);
+  box.getSize(tmpVector);
+  MediaContentBounds.bounds[eid].set(tmpVector.toArray());
```

Comment:
The behavior is changed here in case media is failed to be loaded. Without this change `MediaContentBounds` is not set (because `MediaLoaded` is not set) whilt with this change it is set. Is this change intentional? (Honestly I'm not so sure about `MediaContentBounds`.